.github/workflows/linux-ci.yml

```diff
@@ -50,11 +52,10 @@ jobs:
       - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # pin v3.0.2
 
       - name: Install APT packages
-        run: |
-          # Only libfuse3 is available on ubuntu-22.04,
-          # and `desktop-file-utils` provides `update-desktop-database` used by `tests/scripts/run_testenv.sh`
-          # and a specific version of postgresql is used
-          sudo apt-get -y install libfuse2 fuse desktop-file-utils postgresql-${{ env.PGVERSION }}
+        # Only libfuse3 is available on ubuntu-20.04,
+        # and `desktop-file-utils` provides `update-desktop-database` used by `tests/scripts/run_testenv.sh`
+        # and a specific version of postgresql is used
```

Comment:
I would prefer the comment above `- name: Install ...`

Reply:
I guess it's a matter of opinion :shrug:

Reply:
I guess it's a matter of opinion :shrug: